src/plugins/python/pythonplugin.cpp

```diff
@@ -65,10 +71,16 @@ PythonPlugin::~PythonPlugin()
 {
     for (const ScriptEntry &script : mScripts) {
         Py_DECREF(script.module);
-        Py_DECREF(script.mapFormat->pythonClass());
+        if(script.mapFormat) {
+          Py_DECREF(script.mapFormat->pythonClass());
+        }
+        if(script.tilesetFormat) {
+          Py_DECREF(script.tilesetFormat->pythonClass());
+        }
```

Comment:
Please slightly adjust the coding style `if (` and with 4 spaces for indentation.